### PR TITLE
Add `.github/` folder

### PR DESCRIPTION
**Pull request by yeskunall**
```shell
mv CONTRIBUTING.md && mkdir .github \
&& ./.github/CONTRIBUTING.md
```

`ISSUE_TEMPLATE.md` and `PULL_REQUEST_TEMPLATE.md` taken from [github-issue-templates](https://github.com/stevemao/github-issue-templates).

LMK if there needs to be any changes.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<!--- Provide a general summary of your changes in the Title above -->
```
Comment (mfix22) [inline .github/PULL_REQUEST_TEMPLATE.md]: For both of these, can we make them just very simple? In my opinion, we just need to ask for a screenshot if it is applicable to your PR or issue. Other than that our issues and PRs are pretty simple.

Comment (yeskunall) [reply .github/PULL_REQUEST_TEMPLATE.md]: Sure thing. 👍

**Comment by yeskunall** [comment]
@mfix22 I might have made them _too_ simple. You're free to push to my branch and make changes as you see fit. :ok_man:

**Comment by mfix22** [comment]
I like this better.  cc +@jakedex @briandennis

**Comment by jakedex** [comment]
This is perfect. Thanks, @yeskunall!

**Pull request merged**